devday/attendee/management/commands/remove_inactive_users.py

```diff
@@ -0,0 +1,50 @@
+"""
+Implement a management command to remove inactive users that exist longer than
+the activation period.
+"""
+from datetime import timedelta
+
+from django.conf import settings
+from django.core.management import BaseCommand
+from django.utils import timezone
+
+from attendee.models import DevDayUser
+
+
+class Command(BaseCommand):
+    help = (
+        "Remove all inactive users that are older than the configured ACTIVATION_DAYS"
+    )
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--dry-run",
+            action="store_true",
+            dest="dry_run",
+            default=False,
+            help="Do not delete users",
+        )
+
+    def handle(self, *args, **options):
+        now = timezone.now()
+        cutoff_date = now - timedelta(days=settings.ACCOUNT_ACTIVATION_DAYS)
+
+        if options["verbosity"] > 1:
+            self.stdout.write(
+                "delete inactive users that are older than {}".format(cutoff_date)
+            )
+
+        users = DevDayUser.objects.filter(
+            is_active=False, date_joined__lte=cutoff_date
+        ).order_by("email")
```

Comment:
Why order by email? I don't think this is relevant here.
```suggestion
        )
```

Reply:
just to make the output more useful for human consumption when run with `--dry_run --verbose 2`. There is a database index on the email column so there is no measurable performance penalty.